huaweicloud/services/acceptance/eg/resource_huaweicloud_eg_event_stream_test.go

```diff
@@ -0,0 +1,565 @@
+      "security_protocol": "PLAINTEXT",
+    })
+  }
+  sink {
+    name  = "HC.Kafka"
+    kafka = jsonencode({
+      "connection_id": huaweicloud_eg_connection.test.id,
+      "topic": huaweicloud_dms_kafka_topic.test[0].name,
+      "key_transform": {
+        "type": "ORIGINAL"
+      }
+    })
+  }
+  rule_config {
+    transform {
+      type = "ORIGINAL"
+    }
+  }
+  option {
+    thread_num = 3
+
+    batch_window {
+      count    = 10
+      time     = 5
+      interval = 3
+    }
+  }
+}
+`, testAccEventStream_kafkaSource_base(name), name, action)
+}
+
+func TestAccEventStream_rocketMQ(t *testing.T) {
+	var (
+		obj subscriptions.Subscription
+
+		rName = "huaweicloud_eg_event_stream.test"
+		name  = acceptance.RandomAccResourceName()
+
+		rc = acceptance.InitResourceCheck(rName, &obj, getEventStreamFunc)
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			acceptance.TestAccPreCheck(t)
+			acceptance.TestAccPreCheckEG(t)
+			acceptance.TestAccPreCheckEgAgencyName(t)
+		},
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		CheckDestroy:      rc.CheckResourceDestroy(),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccEventStream_rocketMQSource_step1(name, "START"),
+				Check: resource.ComposeTestCheckFunc(
+					rc.CheckResourceExists(),
+					resource.TestCheckResourceAttr(rName, "name", name),
+					resource.TestCheckResourceAttr(rName, "description", "Created by acceptance test"),
+					resource.TestCheckResourceAttr(rName, "status", "RUNNING"),
+					resource.TestCheckResourceAttr(rName, "action", "START"),
+					resource.TestCheckResourceAttrSet(rName, "created_at"),
+					resource.TestCheckResourceAttrSet(rName, "updated_at"),
+					// Source check
+					resource.TestCheckResourceAttr(rName, "source.#", "1"),
+					resource.TestCheckResourceAttr(rName, "source.0.name", "HC.DMS_ROCKETMQ"),
+					// Target check
+					resource.TestCheckResourceAttr(rName, "sink.#", "1"),
+					resource.TestCheckResourceAttr(rName, "sink.0.name", "HC.FunctionGraph"),
+					// Rules check
+					resource.TestCheckResourceAttr(rName, "rule_config.0.transform.0.type", "ORIGINAL"),
+					// Option check
+					resource.TestCheckResourceAttr(rName, "option.0.thread_num", "2"),
+					resource.TestCheckResourceAttr(rName, "option.0.batch_window.0.count", "5"),
+					resource.TestCheckResourceAttr(rName, "option.0.batch_window.0.time", "3"),
+					resource.TestCheckResourceAttr(rName, "option.0.batch_window.0.interval", "2"),
+				),
+			},
+			{
+				Config: testAccEventStream_rocketMQSource_step1(name, "PAUSE"),
+				Check: resource.ComposeTestCheckFunc(
+					rc.CheckResourceExists(),
+					resource.TestCheckResourceAttr(rName, "status", "STOPPED"),
+					resource.TestCheckResourceAttr(rName, "action", "PAUSE"),
+				),
+			},
+			{
+				ResourceName:      rName,
+				ImportState:       true,
+				ImportStateVerify: true,
+				ImportStateVerifyIgnore: []string{
+					"source",
+					"sink",
```

Comment:
if there parameters are not returned by api, should add a description about it in doc

Reply:
yeah